src/components/Dropdown/dropdown.module.scss

```diff
@@ -0,0 +1,60 @@
+@import '@/styles/mediaQueries.scss';
+@import '@/styles/variables.scss';
+@import '@/styles/colors.scss';
+
+.dropdown {
+  position: relative;
+  /* display: inline-block; */
+
+  .dropdownBtn {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    background-color: $white;
+    border: 1.5px solid $black;
+    border-radius: $brad-sm;
+    padding: 0.5rem;
+    padding-top: 0.75rem;
+
+    .arrowBtn {
+      width: 1.3rem;
+      height: 1.3rem;
+
+      &:hover {
+        cursor: pointer;
+      }
+    }
+  }
+
+  .radiusNull {
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+
+  .dropdownContent {
+    position: absolute;
+    width: 100%;
+    height: 40rem;
+    overflow: scroll;
+    background-color: $white;
+    color: $black;
+    font-size: 0.85rem;
```

Comment:
Didn't talk about it really, but please stick to uniform font-sizes.
Font sizes go in 0.125 steps, so 0.5, 0.6125, 0.75, 0.875, 1 and so on.